modules/swagger-annotations/src/main/java/io/swagger/oas/annotations/media/Schema.java

```diff
@@ -39,192 +35,230 @@
 @Inherited
 public @interface Schema {
   /**
-   * provides a java class as implementation for this schema.  When provided, additional information in the Schema annotation will augment the java class after introspection
+   * Provides a java class as implementation for this schema.  When provided, additional information in the Schema annotation will augment the java class after introspection.
+   * @return a class that implements this schema
    **/
   Class<?> implementation() default Void.class;
 
   /**
-   * provides a java class to be used to disallow matching properties.
+   * Provides a java class to be used to disallow matching properties.
+   * @return a class with disallowed properties
    **/
   Class<?> not() default Void.class;
 
   /**
-   * provides an array of java class implementations which can be used to describe multiple acceptable schemas.  If more than one match the derived schemas, a validation error will occur
+   * Provides an array of java class implementations which can be used to describe multiple acceptable schemas.  If more than one match the derived schemas, a validation error will occur.
+   * @return the list of possible classes for a single match
    **/
-  Class<?>[] oneOf() default Void.class;
+  Class<?>[] oneOf() default {};
```

Comment:
@frantuma is this OK?  Not sure why we would make this change but I trust your input